src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,60 @@
 import React from 'react';
-
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  movie: Movie
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => {
+  const {
+    title,
+    description,
+    imgUrl,
+    imdbUrl,
+    imdbId,
+  } = movie;
+
+  return (
+    <div className="card" data-cy="Movie" key={imdbId}>
```

Comment:
You already provided key to this component in `MovieList`
Checklist p2 and p3.

![image](https://github.com/mate-academy/react_movies-list/assets/84296041/0c7112f3-2e77-4a49-a577-d0316295e8fb)
![image](https://github.com/mate-academy/react_movies-list/assets/84296041/9bcb781d-35e8-4787-9c28-4a62de6bf688)